src/static/js/main.js

```diff
@@ -1,198 +1,39 @@
-            '<input type="email" name="cr-password" id="cr-email2_2356852120719850" placeholder="" class="input-large left">' +
-            '<p>Include letters and numbers, at least 8 characters</p></div>' +
-            '</div>',
-    button: 'Create an account and save progress'
-  } );
-
-  var dom = modal.getDom().container;
-  var tab = dom.querySelector( '#log-in-btn' );
-  tab.addEventListener( 'click', function() {
-    modal.destroy();
-    _showLoginTab();
-  } );
-}
-
-function _showLoginTab() {
-  var modal = _showModal( {
-    header: 'Create an account so you can continue later',
-    body:   '<p>Log in or create an account and we will send you a link and a complaint ID number that you can use to finish submitting this complaint.</p>' +
-            '<div id="save-options" class="select-product-or-issue select-save-method">' +
-            '<label class="radio block save-options-label">' +
-            '<input id="create-account-btn" name="radio_" type="radio" class="radio_input product save-options-radio save-with-email selected" value="save-with-email"/>' +
-            'Create an account <br /></label>' +
-            '<label class="radio block save-options-label active">' +
-            '<input name="radio_" type="radio" class="radio_input save-options-radio product save-with-login" value="save-with-login" checked/>' +
-            'Log in <br /></label> </div> <hr class="save-method-hr" />' +
-            '<div id="save-with-login" class="save-method">' +
-            '<p>If you have submitted a complaint to the CFPB in the past, log in to save this complaint to your account. </p>' +
-            '<div class="span3 cr-label cr-save-option email"><label for="con1-email">Email address or phone number <div class="is-required">Answer Required</div></label>' +
-            '<input type="email" name="cr-email2" id="cr-email2_2356852120719850" placeholder="" class="input-large left"></div>' +
-            '<div class="span3 cr-label cr-save-option"><label for="con1-email">Password <div class="is-required">Answer Required</div></label>' +
-            '<input type="email" name="cr-password" id="cr-email2_2356852120719850" placeholder="" class="input-large left"></div>' +
-            '</div>',
-    button: 'Log in and save progress'
-  } );
-
-  var dom = modal.getDom().container;
-  var tab = dom.querySelector( '#create-account-btn' );
-  tab.addEventListener( 'click', function() {
-    modal.destroy();
-    _showCreateTab();
-  } );
-}
+var globalModals = require( './modules/global-modals' );
+var stepWelcome = require( './modules/step-welcome' );
+var stepSelectProduct = require( './modules/step-select-product' );
+var stepSelectIssue = require( './modules/step-select-issue' );
+var stepWhatHappened = require( './modules/step-what-happened' );
+var stepCompanyInformation = require( './modules/step-company-information' );
+var stepYourInformation = require( './modules/step-company-information' );
```

Comment:
Should this be:

```js
var stepYourInformation = require( './modules/step-your-information' );
```

Also, it looks like that module has a "2" at the end of its file name right now: `step-your-information 2.js`